src/main/java/io/codelex/flightplanner/repository/AirportRepository.java

```diff
@@ -1,35 +1,9 @@
 package io.codelex.flightplanner.repository;
 
 import io.codelex.flightplanner.model.Airport;
-
-import lombok.Synchronized;
+import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
 
-import java.util.ArrayList;
-import java.util.List;
-
 @Repository
-public class AirportRepository {
-    private final List<Airport> airports = new ArrayList<>();
-
-    @Synchronized
-    public void add(Airport airport) {
-        if (!airports.contains(airport)) {
-            airports.add(airport);
-        }
-    }
-
-    public List<Airport> find(String search) {
-        return airports.stream()
-                       .filter(airport -> airport.getAirport().toLowerCase().contains(search)
-                               || airport.getCity().toLowerCase().contains(search)
-                               || airport.getCountry().toLowerCase().contains(search))
-                       .toList();
-    }
-
-    @Synchronized
-    public void clear() {
-        airports.clear();
-    }
-
+public interface AirportRepository extends JpaRepository<Airport, String> {
```

Comment:
un šeit es noteikti neizmantotu String :smile:

Reply:
Vai vienmēr ir nepieciešams numerisks ID, ja Lidostas kods ir unikāls varchar(3)?

Reply:
nav tāds ka gluži `nepieciešams`, built in auto generator always will be less error prone 
un pie zināma apjoma jau ir kaut kāda runa apr performance, long will win string at that :smile:
šajā izmēra jau protams nav liela šķirba, just a better practice